tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/binaryoperations/BitwiseShiftOperationTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.test.expressions.binaryoperations;
+
+import org.ballerinalang.test.BAssertUtil;
+import org.ballerinalang.test.BCompileUtil;
+import org.ballerinalang.test.BRunUtil;
+import org.ballerinalang.test.CompileResult;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/**
+ * Test functionality of bitwise left shift, bitwise right shift and bitwise unsigned right shift operations.
+ *
+ * @since 2.0.0
+ */
+public class BitwiseShiftOperationTest {
```

Comment:
I think we already have tests under different tests. Shall we move them all to one file? We can do this separately.